src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -264,6 +264,10 @@ private static Request ToRequest(FileConfig fileConfig, string path)
             Capture[] c = CFamilyHelper.Capture.ToCaptures(fileConfig, path, out string cfamilyLanguage);
             var request = MsvcDriver.ToRequest(c);
             request.CFamilyLanguage = cfamilyLanguage;
+            if (fileConfig.ItemType == "ClInclude")
+            {
+                request.Flags |= Request.MainFileIsHeader;
```

Comment:
Assert that this flag is set on the returned request in `CreateRequest_HeaderFile_IsSupported` to avoid regression

Reply:
👍
Done.